yoti_python_sdk/tests/dynamic_sharing_service/policy/test_dynamic_policy_builder.py

```diff
@@ -0,0 +1,119 @@
+from yoti_python_sdk.dynamic_sharing_service.policy.dynamic_policy_builder import (
+    DynamicPolicyBuilder,
+)
+from yoti_python_sdk.dynamic_sharing_service.policy.wanted_attribute_builder import (
+    WantedAttributeBuilder,
+)
+
+from yoti_python_sdk import config
+
+
+def test_an_attribute_can_only_exist_once():
+    NAME = "Test name"
+
+    wanted_attribute = WantedAttributeBuilder().withName(NAME).build()
+
+    policy = (
+        DynamicPolicyBuilder()
+        .with_wanted_attribute(wanted_attribute)
+        .with_wanted_attribute(wanted_attribute)
+        .build()
+    )
+
+    assert len(policy["wanted"]) == 1
+    assert wanted_attribute in policy["wanted"]
+
+
+def test_remember_me():
+    policy = DynamicPolicyBuilder().with_wanted_remember_me().build()
+
+    assert policy["wanted_remember_me"]
+
+
+def test_build_with_simple_attributes():
+    builder = DynamicPolicyBuilder()
+    builder.with_family_name()
+    builder.with_given_names()
```

Comment:
Do you think it looks better with the backslash on the preceding line rather than `builder`?
```suggestion
    .with_given_names()\
```

Reply:
What do you think about this one @emmas-yoti?

Reply:
Marking as resolved, as escaping line breaks isn't very Python-esque